server/api/resolvers/requestGroupResolvers.ts

```diff
@@ -130,7 +130,7 @@ const requestGroupResolvers = {
             });
         });
     },
-    nextRecipient: async (parent, __, { authenticateUser }): Promise<String> => {
+    nextRecipient: async (parent, __, { authenticateUser }): Promise<string> => {
```

Comment:
Need to add a resolver for `donationForms`.
See to resolver for `requestTypes` in this file for reference